app/scripts/actions/index.js

```diff
@@ -303,6 +329,34 @@ export const applyWorkflowToGranule = (granuleId, workflow) => ({
   }
 });
 
+export const applyRecoveryWorkflowToGranule = (granuleId) => {
```

Comment:
Just trying to follow the logic of what this action is doing:

1. Get the granule by ID
2. Get collection for that granule
3. Get the granule recovery workflow from the collection, if any
4. Dispatch the recovery workflow on the granule or dispatch error

Correct?

Reply:
You've got it :100: